parm/config/gfs/config.prepoceanobs

```diff
@@ -6,6 +6,7 @@ echo "BEGIN: config.prepoceanobs"
 
 export OCNOBS2IODAEXEC=${HOMEgfs}/sorc/gdas.cd/build/bin/gdas_obsprovider2ioda.x
 
+export SOCA_INPUT_FIX_DIR=@SOCA_INPUT_FIX_DIR@
```

Comment:
https://github.com/guillaumevernieres/global-workflow/blob/feature/soca-fix-to-oceanobsprep/parm/config/gfs/yaml/defaults.yaml will need an update repeating [Line 36](https://github.com/guillaumevernieres/global-workflow/blob/44e00542bd7f8b8703bb77137ef46ad8183d380e/parm/config/gfs/yaml/defaults.yaml#L36) in the [prepoceanobs](https://github.com/guillaumevernieres/global-workflow/blob/44e00542bd7f8b8703bb77137ef46ad8183d380e/parm/config/gfs/yaml/defaults.yaml#L43) section.